src/Collapsible/Collapsible.scss

```diff
@@ -1,56 +1,43 @@
-// Local Variables
-$collapsible-border: 1px solid silver;
```

Comment:
Let's do our best to remove this file entirely and fold the core scss into /core and edx specific scss into edx/theme

Reply:
To be done in the future